src/components/DatePicker/DatePicker.js

```diff
@@ -47,9 +47,15 @@ export default class DatePicker extends Component {
     dateFormat: PropTypes.string,
 
     /**
-     * The value of the `<input>`.
+     * The value of the date value provided to flatpickr, could
+     * be a date, a date number, a date string, an array of dates.
      */
-    value: PropTypes.string,
+    value: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.array,
```

Comment:
Wondering if `PropTypes.arrayOf()` can be used here...?

Reply:
done